emuu-client/src/components/UserAuthentication/newRegister.js

```diff
@@ -34,9 +34,66 @@ function Register() {
 
   const videoCollectionData = {};
 
+    const validatePassword= (pass)=>{
+    if(pass.length < 8){
+    document.querySelector("#password-err").innerHTML = "At least 8 characters";
+    document.querySelector("#password-err").style.color = "red";
+    return false;
+
+    }
+
+    let uppercase = false;
+    let specialChar = false;
+        for( let i = 0 ; i< pass.length; i++){
+    if(pass[i].charCodeAt(0) >= 65 && pass[i].charCodeAt(0)<= 90) uppercase = true;
+    }
+    if(!uppercase) {
+                       document.querySelector("#password-err").innerHTML = "At least 1 uppercase letter"
+                        document.querySelector("#password-err").style.color = "red";
+                       return false;
+
+                       }
+    for( let i = 0 ; i< pass.length; i++){
+        if(pass[i].charCodeAt(0) >= 33 && pass[i].charCodeAt(0)<= 64) specialChar = true;
+        }
+       if(!specialChar) {
+                            document.querySelector("#password-err").innerHTML = "At least 1 special character"
+                             document.querySelector("#password-err").style.color = "red";
+                            return false;
+
+                            }
+
+       return true;
+
+    }
+
+
+    function validateEmail(email)
+   {
+
+    let period = false;
+    let at = false;
+    for( let i=0 ; i< email.length; i++){
+    if(email[i]== '@')at = true;
+    if(email[i]== '.')period = true;
+    }
```

Comment:
Also, you should implement different types of email services, for ex., Must require: Gmail, Yahoo, etc. This will prevent users from just doing Test@.com as an email input